charts/matrix-recorder/values.yaml

```diff
@@ -1,7 +1,7 @@
 environment: production
 
 replicaCount: 1
-domain: archive
+domain: archive.web3.foundation
```

Comment:
This won't work, I think we should have here just `web3.foundation` and change the helper here https://github.com/w3f/matrix-recorder/blob/master/charts/matrix-recorder/templates/_helpers.tpl#L68 to return `archive.{{ .Values.domain }}`, the helper is what we actually use in the ingress and in the certificate.